README.md

```diff
@@ -157,6 +157,8 @@ webClient.registerCallback(new ResultCallback<AuthorizationStatus, Authorization
 
 The `client` can now be used to authenticate users and authorize access.
 
+### Discovery URI Guidance
```

Comment:
Thoughts on changing up the format and adding a link to https://developer.okta.com/docs/concepts/auth-servers/ as well?

Reply:
I'm thinking it also might be good to be more prescriptive here. Maybe if you're using the default authorization server, your url should look like `https://{yourOktaDomain}`, otherwise it should look like `https://{yourOktaDomain} /oauth2/${authServerId}`, and list the other options as alternatives.

Reply:
Good idea. The link is very helpful in explaining the difference.